### ISSUE TITLE
test pretrain model 

### ISSUE THREAD

**Issue by billhyde** [mahyarnajibi/SSH#30]
hi, I tested one picture based on your pretrain model and compare the result with Tiny Face Detector, CVPR 2017(https://github.com/peiyunh/tiny). In your paper, the performance is better than Tiny face,but it seems that the performance is not better than Tiny face. I am wonder if  anything wrong with the way I use the model Or something I need pay more attention when I use the model? Thank you very much.

**Comment by billhyde** [comment]
![1](https://user-images.githubusercontent.com/17793530/41230187-6a43ae2c-6db1-11e8-9742-3e0ede5fa120.jpg)

**Comment by mahyarnajibi** [comment]
Hi,
The main difference is that the demo script provided in the tiny repo is tweaked for this image. As mentioned in the tiny demo file, the hyper-parameters are set differently for this image and using the same parameters result in a lower performance on the WIDER test set.

In general, I think the performance of detectors should not be compared based on a single image as depending on where in the AP curve you set parameters to, the detections can be very different. If you are interested in comparing detectors on a different data other than the standard datasets,  you need a larger unbiased set to have a reasonable error estimator.
Hope this helps!

**Comment by billhyde** [comment]
Thank you for your reply and I will try.

**Comment by mahyarnajibi** [comment]
Sure! Feel free to re-open the issue if you face any problems.
